### PR TITLE
Rephrase

### LINKED ISSUE #5732: Rephrase
The following sentence under LINQ and lambdas section can be rephrased for better clarity:

"Using Language-Integrated Query (LINQ) and lambda expressions is a great example of using productive features that you might later find you need to rewrite if the code becomes a significant impact on performance."

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 01b87b64-6ecb-c03a-12a2-fac0d09e164e
* Version Independent ID: 8348b9cb-fe5f-cc9b-ae70-d2d61a10b530
* Content: [Writing Large, Responsive .NET Framework Apps](https://docs.microsoft.com/en-us/dotnet/framework/performance/writing-large-responsive-apps)
* Content Source: [docs/framework/performance/writing-large-responsive-apps.md](https://github.com/dotnet/docs/blob/master/docs/framework/performance/writing-large-responsive-apps.md)
* Service: **unspecified**
* Product: **dotnet-framework**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by sputier**
This PR contains a reformulation proposal for the #5732 issue

Fixes #5732

### CODE REVIEW

File: docs/framework/performance/writing-large-responsive-apps.md [hunk 1]
```diff
@@ -271,7 +271,7 @@ private static string GetStringAndReleaseBuilder(StringBuilder sb)
  This simple caching strategy adheres to good cache design because it has a size cap.  However, there is more code now than in the original, which means more maintenance costs.  You should adopt the caching strategy only if you’ve found a performance problem, and PerfView has shown that <xref:System.Text.StringBuilder> allocations are a significant contributor.  
   
 ### LINQ and lambdas  
- Using Language-Integrated Query (LINQ) and lambda expressions is a great example of using productive features that you might later find you need to rewrite if the code becomes a significant impact on performance.  
+Language-Integrated Query (LINQ), in conjunction with lambda expressions, is a great example of a productive feature. However, its use may have a significant impact on performance over time, and you might find you need to rewrite your code.
```
Comment (rpetrusha) [inline docs/framework/performance/writing-large-responsive-apps.md]: productive --> productivity
I think that I'd delete "great"; simply an example of a productivity feature

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for addressing the issue, @sputier. I've left a couple of comments about suggested revisions.

File: docs/framework/performance/writing-large-responsive-apps.md (hunk 1 above)
Comment (sputier) [reply docs/framework/performance/writing-large-responsive-apps.md]: Hi @rpetrusha ! Thanks for your remarks. I just pushed a PR to address them.

**Review by rpetrusha — APPROVED** [review]
Thanks, @sputier, for making the additional change. I'll merge your PR now.

**Pull request merged**